content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md

```diff
@@ -0,0 +1,193 @@
+
+10. Make the choice of updating the worker pool in parallel, if required. Click on **Next**.
+
+
+11. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](/clusters/public-cloud/aws/architecture#spotinstances), disk size, and the number of nodes.
+
+<InfoBox>
+
+You can add new worker pools if you need to customize certain worker nodes to run specialized workloads. As an example, the default worker pool may be configured with the m3.large instance types for general-purpose workloads, and another worker pool with instance type g2.2xlarge can be configured to run GPU workloads.
+
+</InfoBox>
+
+ 
+
+12. An optional taint label can be applied to a node pool during the cluster creation. For a an existing cluster, the taint label can be edited, review the [Node Pool](/clusters/cluster-management/node-pool) management page to learn more. Toggle the **Taint** button to create a label.
+
+
+13. Enable or disable node pool taints. If tainting is enabled then you need provide values for the following parameters:
+    
+    |**Parameter**| **Description**|
+    |-------------|---------------|
+    |**Key**      |Custom key for the taint.|
+    |**Value**    | Custom value for the taint key.|
+    | **Effect**  | Make the choice of effect from the drop-down menu. Review the effect table bellow for more details. |
+  
+    #### Effect Table
+    
+    |**Parameter**| **Description**|
+    |-------------|---------------|
+    | **NoSchedule**|  A pod that cannot tolerate the node taint and should not be scheduled to the node. 
+    | **PreferNoSchedule**| The system will avoid placing a non-tolerant pod to the tainted node but is not guaranteed.
+    | **NoExecute**|  New pods will not be scheduled on the node, and existing pods on the node if any on the node will be evicted they do not tolerate the taint. |
+
+14. Click on **Next**.  
+    
+15. The settings page is where you can configure patching schedule, security scans, backup settings, setup role based access control (RBAC), and enable [Palette Virtual Clusters](/devx/palette-virtual-clusters). Review the settings and make changes if needed. Click on **Validate**.
+
+16. Review the settings summary and click on **Finish Configuration** to deploy the cluster. Be aware that provisioning IaaS clusters can take several minutes.
+
+The cluster details page of the cluster contains the status and details of the deployment. Use this page to track the deployment progress.
+
+
+# Validate
+
+You can validate your cluster is up and running by reviewing the cluster details page. Navigate to the left **Main Menu** and click on **Clusters**. The **Clusters** page contains a list of all available clusters managed by Palette. Click on the row for the cluster you wish to review its details page. Ensure the **Cluster Status** field contains the value **Running**.
```

Comment:
`Reviewing the cluster details page, you can validate that your cluster is up and running. Navigate to the left **Main Menu** and click on **Clusters**. The **Clusters** page contains a list of all available clusters managed by Palette. Click on the row for the cluster you wish to review its details page. Ensure the **Cluster Status** field contains the value **Running**.
		`